marlowe-dashboard-client/src/Welcome/State.purs

```diff
@@ -134,14 +134,14 @@ handleAction (OpenPickupWalletCardWithDetails walletDetails) = do
       handleAction $ WalletNicknameOrIdInputAction $ InputField.Reset
       handleAction $ WalletNicknameInputAction $ InputField.SetValue $ view _walletNickname walletDetails
       handleAction $ WalletIdInputAction $ InputField.SetValue $ UUID.toString (unwrap (view _companionAppId walletDetails))
-      handleAction $ OpenCard PickupWalletCard
+      handleAction $ OpenCard ConnectWalletCard
 
 handleAction (WalletNicknameInputAction inputFieldAction) = toWalletNicknameInput $ InputField.handleAction inputFieldAction
 
 handleAction (WalletIdInputAction inputFieldAction) = toWalletIdInput $ InputField.handleAction inputFieldAction
 
-handleAction (PickupWallet walletNickname) = do
-  assign _pickingUp true
+handleAction (ConnectWallet walletNickname) = do
```

Comment:
I was a little disorianted with `_connecting` until I saw this. Why ConnectWallet and not UseWallet? What are we connecting?

Reply:
Yeah, it probably should be `UseWallet`. It's because before we had "Pickup" on the button and a "pickingUp" flag in the state, and "Picking up..." on the button when `pickingUp == true`. But "using" in that case doesn't make sense. Anyway, I'll try and think of something better and push it shortly.